commercetools-sdk-base/src/main/java/io/sphere/sdk/queries/PagedResult.java

```diff
@@ -50,6 +50,22 @@
      }
 
     /**
+     * calculates the page number of the result.
+     *
+     * @return the page number of the result.
+     */
+    default long getPage() {
+
+        if (getOffset() == null) {
+            throw new UnsupportedOperationException("Can only be used if the offset is known.");
+        }
+        if (getCount() == null || getCount() <= 0) {
```

Comment:
👍 I am ok with assumption that the null check should be removed, still we should keep the check that getCount() is strictly positive since you can request paged query with 0 as a size.
in that case the page concept doesn't make sense.